client/src/pages/Login/Login.tsx

```diff
@@ -36,25 +42,45 @@ export default function Login(): JSX.Element {
     });
   };
 
+  //---CODE FOR ADDING A BUTTON FOR A DEMO USER: DEVELOPMENT ONLY
+  const handleDemoUser = () => {
+    login('test@test.com', 'test123').then((data) => {
+      if (data.success) {
+        updateLoginContext(data.success);
+      }
+    });
+  };
+
   return (
-    <Grid container component="main" className={classes.root}>
+    <Grid container component="main" direction="row" wrap="nowrap" className={classes.root}>
+      <Logo />
       <CssBaseline />
       <Grid item xs={12} sm={8} md={7} elevation={6} component={Paper} square>
         <Box className={classes.authWrapper}>
-          <AuthHeader linkTo="/signup" asideText="Don't have an account?" btnText="Create account" />
           <Box width="100%" maxWidth={450} p={3} alignSelf="center">
             <Grid container>
               <Grid item xs>
                 <Typography className={classes.welcome} component="h1" variant="h5">
-                  Welcome back!
+                  Log in
                 </Typography>
               </Grid>
             </Grid>
             <LoginForm handleSubmit={handleSubmit} />
           </Box>
-          <Box p={1} alignSelf="center" />
+          <Box alignSelf="center">
+            <Typography variant="button">
+              Not a member yet?
+              <span> </span>
```

Comment:
Is this here to separate the text and the link? Could this be refactor to be a bit more cleaner?

Reply:
I've modified this to use inline CSS to add a margin to the text